Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM eclipse-temurin:19-jre-alpine
+FROM eclipse-temurin:19-jre-jammy
 
 USER root
-RUN apk add --no-cache graphviz
+RUN apt update && apt install graphviz --yes
```

Comment:
It’s also a good idea to clear the apt cache afterwards, see eg https://techoverflow.net/2021/01/13/how-to-use-apt-install-correctly-in-your-dockerfile/

Reply:
@jp7677 added. Thanks for tip, I was bit hesitant if this should be `rm -rf` or `apt clean` but stayed with your suggestion.